package.json

```diff
@@ -20,6 +20,7 @@
     "infusion": "2.0.0",
     "kettle": "1.6.4",
     "sjrk-storytelling": "https://github.com/waharnum/sjrk-storyTelling#FLOE-525",
+    "couch-config": "https://github.com/waharnum/couch-config#SJRK-6",
```

Comment:
Can you also update `README.md` to remove the mention of `stories.json` and add some detail about the Couch configuration usage?